docker/policystream

```diff
@@ -36,10 +37,27 @@ RUN . /usr/local/bin/activate && \
 
 RUN mkdir /output
 
+# Update git since the base version does not allow for --initial-branch
```

Comment:
afaict an apt install should be cool here since we updated the default base image to 22.04 across the board in #7619. (22.04 gives us git 2.34.1 where 20.04 was giving us git 2.25.1, and `--initial-branch` needed >= 2.28)